### PR TITLE
Removes juicers

### PR DESCRIPTION
**Pull request by ACCount12**
They are not constructable/deconstructable, not used on main maps, and inferior to All-In-One-Grinders in every way possible. It's time to trash them.

### CODE REVIEW

**Comment by The-clown-known-as-Pepe** [comment]
You're making a mistake

This is the only thing stopping fruit from taking over.

**Comment by DaxDupont** [comment]
But the day of the juicer!

**Comment by ACCount12** [comment]
We'll have to fight the tyranny of oranges with grinders now.

**Comment by Tlaltecuhtli** [comment]
just remove the grinders juice function

File: code/modules/food_and_drinks/kitchen_machinery/juicer.dm [hunk 1]
```diff
@@ -1,187 +0,0 @@
...
-		return
-	playsound(src.loc, 'sound/machines/juicer.ogg', 50, 1)
-	for (var/obj/item/reagent_containers/food/snacks/O in src.contents)
-		var/r_id = get_juice_id(O)
-		beaker.reagents.add_reagent(r_id,get_juice_amount(O))
-		qdel(O)
-		if (beaker.reagents.total_volume >= beaker.reagents.maximum_volume)
-			break
-
-/obj/structure/closet/crate/juice/PopulateContents()
```
Comment (Cyberboss) [inline code/modules/food_and_drinks/kitchen_machinery/juicer.dm]: Keep this

**Review by Cyberboss — APPROVED** [review]

**Comment by BeeSting12** [comment]
DAY OF THE JUICER SOON

**Comment by arsserpentarium** [comment]
You are slowpokes. There are already tyranny of fruits. They just removes juicers to solidify their position.

**Comment by banthebantz** [comment]
>tyranny

brainwashed

**Comment by Armhulen** [comment]
I wasn't that much against this PR, but OP appears to be oranges. That earns a solid 👎  from me.

**Comment by WJohn** [comment]
Can we also combine juicing/grinding? There's no reason to have both options when really they should do the same job of "turn object into just its reagents".

**Comment by ACCount12** [comment]
I believe there are differences, and this is out of scope for this PR.

**Comment by arsserpentarium** [comment]
I like idea of Wjohn. I don't like separate juicing/grinding for several reasons.
But logically juicing=grinder+filter with pressure. So maybe there is point to refactor juicing into reagent conversion.

**Pull request merged**